src/components/ui/PageHead.tsx

```diff
@@ -11,6 +16,7 @@ export type PageHeadProps = {
   additionMeta: Nullable<ReactElement>;
   currentArticleMeta: Nullable<TutorialMeta | BlogArticleMeta>;
   commitMeta: Nullable<CommitMeta>;
+  jsonLd?: Nullable<BlogArticleJsonLD>;
```

Comment:
@iamnamananand996 In our codebase we want to enforce the explicit props policy. So we try to force every prop not take undefind as their type. 

```suggestion
  jsonLd: Nullable<BlogArticleJsonLD>;
```